version.bzl

```diff
@@ -2,7 +2,7 @@
 MIN_VERSION = "0.28.0"
```

Comment:
`--experimental_allow_incremental_repository_updates` is still needed for 0.28.0